.github/workflows/test_on_push.yaml

```diff
@@ -0,0 +1,39 @@
+name: Run tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-tests-ce:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
```

Comment:
I have the better snippet, which should handle PR to a fork gracefully:

https://github.com/tarantool/smtp/blob/322254c83169905bc43cb8a25972fa19a03f72ea/.github/workflows/testing.yml#L12-L17